data/presets/highway/trunk.json

```diff
@@ -31,6 +32,8 @@
         "highway": "trunk"
     },
     "terms": [
+        "expressway",
+        "highway",
```

Comment:
if we add `highway` here as a term, we should also add it to all other `highway` presets, shouldn't we?

Reply:
The default localization is American English. In a generic American dialect, “highway” only refers to high-speed, limited- or controlled-access roads. It has a broader meaning in some dialects, but “road” or “street” is always preferred for surface streets. Meanwhile, in California, it means `route=road` among the general public and only means `highway=*` among civil engineers.

My understanding is that “highway” means `highway=*` in British English, but iD has an en-GB localization for things like that.

Reply:
OK, that makes sense, thanks!